### PR TITLE
[New Concept Entry for existing topic] POSTGRESQL array-constructor

### LINKED ISSUE #4592: [Concept Entry] Postgresql array-constructors
### Reason/inspiration (optional)

We would like a new entry on the `Array Constructors` concept under PostgreSQL. The entry should go in a new file under `docs/content/postgresql/concepts/array-constructors/array-constructors.md`.

The entry should include:

- An introductory paragraph that describes the concept
- A �Syntax� section that includes the syntax for the concept (if applicable)
- An �Example� section that shows the concept in use

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by achur00**
### Description

I created a new entry and file in the path docs/content/postgresql/concepts/array-constructors/array-constructors.md.

### Issue Solved
Issue Solved

Closes #4592

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/postgresql/concepts/array-constructors/array-constructors.md
Comment (cigar-galaxy82) [inline content/postgresql/concepts/array-constructors/array-constructors.md:10]:
```suggestion
  - 'PostgreSQL'
```

File: content/postgresql/concepts/array-constructors/array-constructors.md
Comment (cigar-galaxy82) [inline content/postgresql/concepts/array-constructors/array-constructors.md:128]:
```suggestion
Now, if we check our `example3` table again we will find out that the contact `(+234)81-1265-3347` has been changed to `(+234)81-0000-0000` in the table below:
```

File: content/postgresql/concepts/array-constructors/array-constructors.md
Comment (cigar-galaxy82) [inline content/postgresql/concepts/array-constructors/array-constructors.md:55]:
```suggestion
-- 'id' is an auto-incrementing primary key, and 'team' can hold a collection of text.
```

File: content/postgresql/concepts/array-constructors/array-constructors.md
Comment (cigar-galaxy82) [inline content/postgresql/concepts/array-constructors/array-constructors.md:72]:
```suggestion
We have now successfully learned how to insert arrays into our postgres table, now let's learn
```

File: content/postgresql/concepts/array-constructors/array-constructors.md
Comment (cigar-galaxy82) [inline content/postgresql/concepts/array-constructors/array-constructors.md:73]:
```suggestion
We have now successfully learned how to insert arrays into our postgres table, now let's learn
how to access our array element using the `example3` table below:
```

**Comment by cigar-galaxy82** [comment]
Hi @achur00  Thank you for contributing to the Codecademy Docs

I've suggested some changes for this entry, could you please review and modify those at your earliest convenience

**Comment by achur00** [comment]
Alright,  will do, thanks.

**Comment by achur00** [comment]
Hello @cigar-galaxy82 I have made all corrections in a new commit, let me know if you have any recommendation.

**Comment by ishg-153** [comment]
@cigar-galaxy82 Please approve this PR.

**Comment by cigar-galaxy82** [comment]
Hi @achur00 thank you now all checks have passed.

**Review by cigar-galaxy82 — APPROVED** [review]

**Comment by achur00** [comment]
@cigar-galaxy82 Do I have something more i need to do before this can be merged?

**Comment by cigar-galaxy82** [comment]
Hi @achur00 I have merged the PR

**Comment by ishg-153** [comment]
@cigar-galaxy82 I wasn't finished with my review. Once the second reviewer is done, then they merge the PR. Please remember this for future PRs.

**Comment by ishg-153** [comment]
@achur00 Are you comfortable opening a new PR to make changes within this entry? The changes are as follows:
- Remove any personal pronouns used like `our`.
- Remove the link from `## Syntax` section as we avoid adding links outside of Codecademy

I will suggest a couple more direct changes to that PR and then it can be merged. When you create a new PR, tag me so we can treat it as a continuation of this PR.

**Comment by cigar-galaxy82** [comment]
Hi @ishg-153 I will make sure this does not happen again.

**Comment by cigar-galaxy82** [comment]
Hi @achur00 I am reverting this PR as the second review was not complete.

**Pull request merged**
